docs/topics/api/abuse.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Abuse Reports
+=============
+
+The following API endpoint covers abuse reporting
+
+-------------------
+Submitting a report
+-------------------
+
+.. _`abusereport-create`:
+
+The following API endpoint allows an abuse report to be submitted for an Add-on
+or user on addons.mozilla.org.  Authentication is not required, but is recommended
+so reports can be responded to if nessecary.
+
+.. http:post:: /api/v3/abuse/report/
+
+    .. _abusereport-create-request:
+
+    :<json string|null addon: if reporting an add-on then the add-on id, slug, or guid of the add-on.
+    :<json string|null user: if reporting a user then the user id or username of the user.
+    :<json string message: An explanation of the reason for the report (required).
+    :>json object|null reporter: The user who submitted the report, if authenticated.
+    :>json int reporter.id: The id of the user who submitted the report.
+    :>json string reporter.name: The name of the user who submitted the report.
+    :>json string reporter.url: The link to the profile page for of the user who submitted the report.
```

Comment:
Actually, thinking further on this - it's going to more straightforward to keep it as is for now and then make the change everywhere at once.  The frontend should be able to trust that if a profile url has been returned it will be accessible.